src/3box.js

```diff
@@ -170,15 +170,18 @@ class ThreeBox {
    * @param     {Object}        opts                    Optional parameters
    * @param     {Object}        opts.ipfsOptions        A ipfs options object to pass to the js-ipfs constructor
    * @param     {String}        opts.orbitPath          A custom path for orbitdb storage
+   * @param     {Function}      opts.consentCallback    A function that will be called when the user has consented to opening the box
    * @return    {ThreeBox}                              the threeBox instance for the given address
    */
   static async openBox (address, web3provider, opts = {}) {
     let muportDID
     let serializedMuDID = localstorage.get('serializedMuDID_' + address)
     if (serializedMuDID) {
       muportDID = new MuPort(serializedMuDID)
+      if (opts.consentCallback) opts.consentCallback('user already logged in')
```

Comment:
do we differentiate here for UI reasons? maybe we pass a more standard val, while very dev readable not super machine readable. Maybe bool, or if eventually more values, some simpler const strings

Reply:
Yeah, this was a bit rushed. I guess a bool would make more sense for now

Reply:
I'm a bit unsure of how to document it though.

Reply:
I'm changing to true/false. Let's document this later.